### PR TITLE
Rename `racket' to `minimal-racket'

### PR DESCRIPTION
**Pull request by leafac**
- [x] Have you followed the guidelines in our [Contributing](https://github.com/Homebrew/homebrew-core/blob/master/.github/CONTRIBUTING.md) document?
- [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally prior to submission with `brew install <formula>` (where `<formula>` is the name of the formula you're submitting)?
- [x] Does your submission pass `brew audit --strict --online <formula>` (after doing `brew install <formula>`)?

---

See https://github.com/caskroom/homebrew-cask/pull/23508#issuecomment-238556262.

### CODE REVIEW

**Comment by MikeMcQuaid** [comment]
Nice work but can you check out these docs: https://github.com/Homebrew/brew/blob/master/share/doc/homebrew/Rename-A-Formula.md

Thanks!

**Comment by leafac** [comment]
@MikeMcQuaid: Thanks for the quick feedback. I read the documentation you linked; added an entry to `formula_renames.json` and fixed the commit message.

Also, I recommend adding a link to that on the Pull Request template. Maybe under a section “Rename a Formula”, like the [Homebrew Cask Pull Request template](https://github.com/caskroom/homebrew-cask/blob/8b53b4e9473b9dde6d30b22dc86d5368ead06195/.github/PULL_REQUEST_TEMPLATE.md) is organized in sections.

File: formula_renames.json [hunk 1]
```diff
@@ -24,5 +24,6 @@
   "plt-racket": "racket",
   "polarssl": "mbedtls",
   "screenbrightness": "brightness",
-  "wires": "geckodriver"
+  "wires": "geckodriver",
+  "racket": "minimal-racket"
```
Comment (MikeMcQuaid) [inline formula_renames.json]: Sorry for pedantry but can you put this in alphabetical order? Thanks!

**Comment by MikeMcQuaid** [comment]
May also want to make an alias for `racket`

File: formula_renames.json (hunk 1 above)
Comment (jawshooah) [inline formula_renames.json]: The rest of the file doesn't seem to be sorted 😞

Comment (leafac) [inline formula_renames.json]: @MikeMcQuaid, @jawshooah: Thank you for the feedback.

@jawshooah correctly pointed out that the file currently is not in order. So you want me to sort the whole thing? Do I sort in terms of the old name (the key) or the new one (the value)? Shouldn't I do it in a separate Pull Request?

**Comment by leafac** [comment]
@MikeMcQuaid: I think having an alias for `racket` defeats the purpose of https://github.com/caskroom/homebrew-cask/pull/23508.

**Comment by MikeMcQuaid** [comment]
@leafac Fair enough 👍.

**Comment by MikeMcQuaid** [comment]
Thanks for your contribution to Homebrew! Without people like you submitting PRs we couldn't run this project. You rock!

File: formula_renames.json (hunk 1 above)
Comment (MikeMcQuaid) [inline formula_renames.json]: Sorted for next time in 2e4af01.

**Comment by leafac** [comment]
@MikeMcQuaid: That _you_ for helping maintain Homebrew 👍
